module_text_llm/module_text_llm/helpers/models/openai.py

```diff
@@ -90,8 +90,8 @@ def _use_openai_credentials():
     openai.api_version = None
 
 
-openai_available = os.environ.get("LLM_OPENAI_API_KEY") is not None
-azure_openai_available = os.environ.get("LLM_AZURE_OPENAI_API_KEY") is not None
+openai_available = len(os.environ.get("LLM_OPENAI_API_KEY") or "") > 0
+azure_openai_available = len(os.environ.get("LLM_AZURE_OPENAI_API_KEY") or "") > 0
```

Comment:
I really like how explicit this is, so this is a purely optional suggestion:
```suggestion
openai_available = bool(os.environ.get("LLM_OPENAI_API_KEY"))
azure_openai_available = bool(os.environ.get("LLM_AZURE_OPENAI_API_KEY"))
```